lib/matplotlib/text.py

```diff
@@ -163,6 +163,40 @@ def __init__(self,
         """
         super().__init__()
         self._x, self._y = x, y
+        self._reset_visual_defaults(
+            text=text,
+            color=color,
+            fontproperties=fontproperties,
+            usetex=usetex,
+            parse_math=parse_math,
+            wrap=wrap,
+            verticalalignment=verticalalignment,
+            horizontalalignment=horizontalalignment,
+            multialignment=multialignment,
+            rotation=rotation,
+            transform_rotates_text=transform_rotates_text,
+            linespacing=linespacing,
+            rotation_mode=rotation_mode,
+            **kwargs,
+        )
+
+    def _reset_visual_defaults(
+        self,
+        text='',
+        color=None,
+        fontproperties=None,
+        usetex=None,
+        parse_math=None,
+        wrap=False,
+        verticalalignment='baseline',
+        horizontalalignment='left',
+        multialignment=None,
+        rotation=None,
+        transform_rotates_text=False,
+        linespacing=None,
+        rotation_mode=None,
+        **kwargs,
+    ):
         self._text = ''
```

Comment:
Move this line to `__init__` (this is just for creating the argument, `set_text` updates it.

Reply:
I also thnik that `self.update` doesn't really belong in `_reset...`

Reply:
Thank you for your review @oscargus ! but I'm a bit comfused about the first comment.
You mean remove `_rest_visual_defaults` in Text should be removed and the following lines (ll 200-) are moved in `__init__` right?

Reply:
Sorry. 

I meant that the lines  `self._text = ''` and `self.update(**kwargs)` should be moved to `__init__`,

so something like:
```
self._text = ''
self._reset_visual_defaults(...)  # without kwargs
self.update(kwargs)
```